File: src/2469-Convert-the-Temperature/convert-the-temperature.cpp
#include <iostream>
#include <vector>

class Solution {
public:
    std::vector<double> convertTemperature(double celsius) {
        double kelvin = celsius + 273.15;
        double farenheit = (celsius * 1.80) + 32.00;

        std::vector<double> ans = {kelvin, farenheit};
        return ans;
    }
};

int main() {
    double celsius = 122.11;

    Solution solution;

    std::vector<double> output = solution.convertTemperature(celsius);
    
    for (auto it : output) {
        std::cout << it << " ";
    }
    return 0;
}

File: src/2469-Convert-the-Temperature/convert-the-temperature.py
from typing import List

class Solution:
    def convertTemperature(self, celsius: float) -> List[float]:
       kelvin: float = celsius + 273.15
       farenheit: float = (celsius * 1.80) + 32.00

       ans: List[float] = [kelvin, farenheit]
       return ans

if __name__ == "__main__":
    celsius: float = 122.11   
    solution = Solution()
    
    print(solution.convertTemperature(celsius))